vulnz/src/main/java/io/github/jeremylong/vulnz/cli/commands/CveCommand.java

```diff
@@ -144,6 +146,69 @@ public Integer timedCall() throws Exception {
             LOG.info("NVD_API_KEY not found. Supply an API key for more generous rate limits");
             apiKey = null;// in case it is empty
         }
+        NvdCveClientBuilder builder = getNvdCveClientBuilder(apiKey);
+
+        if (configGroup != null && configGroup.cacheSettings != null) {
+            CacheProperties properties = new CacheProperties(configGroup.cacheSettings.directory);
+            checkIfHttpCacheExists(properties);
+            if (properties.has("lastModifiedDate")) {
+                ZonedDateTime start = properties.getTimestamp("lastModifiedDate");
+                ZonedDateTime end = start.minusDays(-120);
```

Comment:
Why minus negative days instead of plus positive days? Makes the code harder to read in my opinion.

Reply:
I'll fix this in a follow-on PR as I have other changes in the works as well.